wire/protocol.go

```diff
@@ -39,6 +39,10 @@ const (
 	// service flag.
 	BIP0111Version uint32 = 70011
 
+	// FeeFilterVersion is the protocol version which added a new
+	// feefilter message.
+	FeeFilterVersion uint32 = 70013
```

Comment:
Mind putting this after `SendHeadersVersion` so they are in increasing order?  I also see that `RejectVersion` managed to sneak in out of order too.